codewof/templates/users/dashboard.html

```diff
@@ -30,11 +30,24 @@ <h2>Your questions for today</h2>
             <strong>Sorry!</strong> We currently don't have questions for you to do today.
           </div>
         {% endif %}
+        <p>You've solved {{number_days_used}} questions in the last month!</p>
```

Comment:
![image](https://user-images.githubusercontent.com/25916475/65398698-e2f21300-de0c-11e9-8498-2601640f97e4.png)

"...and earned X points..."  perhaps?